lib/workers/send_user_invitation_email.ex

```diff
@@ -0,0 +1,46 @@
+defmodule ChatApi.Workers.SendUserInvitationEmail do
+  @moduledoc false
+
+  use Oban.Worker, queue: :mailers
+
+  alias ChatApi.{Accounts, Users}
+  alias ChatApi.Repo
+
+  require Logger
+
+  @impl Oban.Worker
+  @spec perform(Oban.Job.t()) :: :ok
+  def perform(%Oban.Job{
+        args: %{
+          "user_id" => user_id,
+          "account_id" => account_id,
+          "to_address" => to_address,
+          "invitation_token" => invitation_token
+        }
+      }) do
+    if send_user_invitation_email_enabled?() do
+      user = Users.find_by_id!(user_id) |> Repo.preload([:profile])
+      account = Accounts.get_account!(account_id)
+      Logger.info("Sending user invitation email to #{to_address}")
+
+      ChatApi.Emails.send_user_invitation_email(
```

Comment:
`send_user_invitation_email` returns a `deliver_result` that might contain `{:error, binary()}`. Should we log the error when it gets returned?